uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -661,6 +661,8 @@ sub in_range
     my ($value, $range_string) = @_;
     chomp($value);
 
+    print "This function is deprecated and is replaced with in_min_max to identify the mri_protocol\n";
```

Comment:
in_range function is being called in other instances where people want to check the range of something so I would not call it deprecated...

Reply:
I did grep the codebase, and there is no occurrence of it except here (unless I missed something?)
There is actually **another** in_range function in `MNI/NumericUtilities.pm` which takes in min max as its arguments, but returns different codes...

but either way is fine for me. I was thinking maybe we send a deprecation messagr to the user now, and use a new function. Then in the next release, really remove the old in_range function...
but am open to whatever everyone thinks is the best way to proceed :)

Reply:
in_range is being used in the following places:
- [MRIProcessingUtility.pm line 514](https://github.com/aces/Loris-MRI/blob/9cf1f5e0f8d6f315a0fb321cf228b6fb6fed6dd4/uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm#L514)  by the extra file checks
- [MRI.pm line 595](https://github.com/aces/Loris-MRI/blob/9cf1f5e0f8d6f315a0fb321cf228b6fb6fed6dd4/uploadNeuroDB/NeuroDB/MRI.pm#L595), although, for this one it is within the function `debug_inrange()` that does not seem to be used anywhere in the code...

My guess is that you probably grepped with the & in front of the in_range, which is probably why you did not see those two instances.

Reply:
mmm, I actually did not use `&`, and I knew about the debug_in_range (which I think can be ignored), BUT the important one is the extra file checks... I did not see this one, so great catch, but now I wonder if this table too should not have ranges then!!!!

Reply:
yes, it would make sense to normalize these ranges too then. Maybe in a separate PR?

Reply:
yes I can do the changes, but it all depends on how it will be done on the Loris side...